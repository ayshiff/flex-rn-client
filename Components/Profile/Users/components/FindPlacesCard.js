import { View } from "react-native";
import { Button } from "react-native-elements";
import I18n from "react-native-i18n";
import React from "react";
import styles from "../../ProfileScreenStyles";

const FindPlacesCard = (props: { users: any }) => {
  const { users } = props;
  return (
    <View style={styles.emptyPlaces_container}>
      <Button
        fontWeight="bold"
        iconRight={{
          name: "sync",
          type: "font-awesome5"
        }}
        large={false}
        borderRadius={15}
        backgroundColor="#2E89AD"
        color="#fff"
        style={styles.free_places}
        title={I18n.t("users.users")}
        onPress={users}
      />
    </View>
  );
};

export default FindPlacesCard;
