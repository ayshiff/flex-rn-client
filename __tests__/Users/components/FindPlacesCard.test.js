import { View } from "react-native";
import React from "react";
import renderer from "react-test-renderer";
import * as enzyme from "enzyme";
import ReactSixteenAdapter from "enzyme-adapter-react-16";
import FindPlacesCard from "../../../Components/Profile/Users/components/FindPlacesCard";

enzyme.configure({ adapter: new ReactSixteenAdapter() });

const users = jest.fn();

it("renders correctly", () => {
  const wrapper = renderer.create(<FindPlacesCard users={users} />).toJSON();
  expect(wrapper).toMatchSnapshot();
});
