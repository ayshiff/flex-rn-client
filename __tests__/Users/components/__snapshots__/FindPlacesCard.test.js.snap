// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": 20,
      "marginLeft": 50,
      "marginRight": 50,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        undefined,
        undefined,
        Object {
          "borderRadius": 15,
        },
      ]
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "marginTop": 20,
        }
      }
    >
      <View
        pointerEvents="box-only"
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#9E9E9E",
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 19,
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Object {
              "backgroundColor": "#fff",
            },
            Object {
              "borderRadius": 15,
            },
            Object {
              "padding": 12,
            },
            undefined,
            undefined,
            undefined,
            Object {
              "borderColor": "#2E89AD",
              "borderWidth": 0.5,
            },
            undefined,
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {},
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Array [
                Object {
                  "color": "white",
                  "fontSize": 20,
                },
                Object {
                  "color": "#2E89AD",
                },
                Object {
                  "fontSize": 17.5,
                },
                undefined,
                undefined,
                Object {
                  "fontWeight": "bold",
                },
                Object {
                  "fontFamily": "Raleway",
                },
                undefined,
              ],
            ]
          }
        >
          [missing "en.users.users" translation]
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#2E89AD",
                "fontSize": 18,
              },
              Array [
                Object {
                  "marginLeft": 10,
                },
                undefined,
              ],
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          type="font-awesome5"
        >
          î˜§
        </Text>
      </View>
    </View>
  </View>
</View>
`;
